docs/version_history.md

```diff
@@ -0,0 +1,5 @@
+# Version history
+
+{%
```

Comment:
`--8<-- CHANGELOG.md` if using [pymdownx.snippets](https://facelessuser.github.io/pymdown-extensions/extensions/snippets/)

Reply:
I tried that, but the `on_page_markdown` hook in `changelog_highlight.py` doesn't work with snippets and I'm not sure why. Possibly because the snippets pre-processor runs after that. So I propose to stick to this as-is for now.